ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ExplainExecutor.java

```diff
@@ -44,14 +44,18 @@ private ExplainExecutor() { }
 
   @SuppressWarnings("unchecked")
   public static Optional<KsqlEntity> execute(
-      final PreparedStatement statement,
+      final PreparedStatement<?> statement,
```

Comment:
would be nice if these _specific_ handlers took the specific types, i.e. 
```suggestion
      final PreparedStatement<Explain> statement,
```

Reply:
This made things kinda ugly everywhere else, but it ended up working out.